packages/create-react-native-library/templates/windows-cpp-view-library/windows/{%- project.name %}/{%- project.name %}ViewManager.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <winrt/Windows.UI.Xaml.Controls.h>
+#include <winrt/Windows.UI.Xaml.h>
+#include <winrt/Microsoft.ReactNative.h>
+
+namespace winrt::<%- project.name %>::implementation
+{
+    class <%- project.name %>ViewManager : public implements<
+        <%- project.name %>ViewManager,
+        Microsoft::ReactNative::IViewManager>
```

Comment:
I think you'll probably want to implement some of the other interfaces, e.g. [IViewManagerWithReactContext](https://microsoft.github.io/react-native-windows/docs/IViewManagerWithReactContext)

Reply:
I wanted to keep the starting point simple so a developer can add whatever will be used, but if you recommend adding this then I'll go with your advice 👍